CHANGES.rst

```diff
@@ -185,6 +185,12 @@ Bug Fixes
   - Fixed a bug where ``get_transform`` could sometimes produce confusing errors
     because of a typo in the input validation. [#5645]
 
+  - Ensured that frame attributes cannot be set on ``SkyCoord`` if the current
```

Comment:
This is pretty confusing, and I think the first part was already true.  What about:

Changed ``SkyCoord`` so that frame attributes which are not valid for the current ``frame`` (but are valid for other frames) are stored on the ``SkyCoord`` instance instead of the underlying ``frame`` instance.  An example is setting ``relative_humidity`` on an ICRS ``SkyCoord`` instance.